tests/unit/lms/views/lti/basic_launch_test.py

```diff
@@ -252,9 +235,16 @@ def test__store_lti_data(
         lti_h_service,
         LtiLaunches,
         grading_info_service,
+        application_instance_service,
     ):
+        application_instance_service.get_current.return_value = Mock()
```

Comment:
Our standard application instance service returns a real application instance, which is kind of a pain here